client/app/common/cloud-download-service.js

```diff
@@ -0,0 +1,13 @@
+class CloudDownload {
+    download (url) {
+        const link = document.createElement("a");
+        link.setAttribute("href", url);
+        link.setAttribute("target", "_blank");
+        link.style.visibility = "hidden";
+        document.body.appendChild(link);
+        link.click();
+        document.body.removeChild(link);
+    }
+}
+
+angular.module("managerApp").service("CloudDownload", CloudDownload);
```

Comment:
Already done in ControllerHelper
downloadUrl and downloadContent

Reply:
That was a miss. Sorry. Using ControllerHelper.downloadUrl now. Thanks :)